File: fontes/c/micro/micro09.c
#include<stdio.h>

int main()
{
 float preco;
 float venda;
 float novo_preco;

 printf("Digite o preço: ");
 scanf("%f", &preco);
 printf("Digite a venda: ");
 scanf("%f", &venda);
 if((venda < 500) || (preco < 30))
 {
 	novo_preco = (preco + 0.1*preco);
 }
 else if((venda >= 500 && venda < 1200) || (preco >= 30 && preco <80))
 {
 	novo_preco = (preco + 0.15*preco);
 }
 else if(venda >= 1200 || preco >= 80)
 {
 	novo_preco = (preco - 0.2*preco);
 }
 printf("O novo preço é: %f", novo_preco);

 return 0;
}

File: fontes/pascal/micro/micro09.pas
program micro09;
 var
 preco,venda,novo_preco : real;

 begin
 write('Digite o preco: ');
 readln(preco);
 write('Digite a venda: ');
 readln(venda);
 if(venda < 500) or (preco < 30) then
 novo_preco := preco + 10/100 * preco
 else if (((vendas >= 500) and (venda < 1200)) or ((preco >= 30) and (preco < 80))) then
 novo_preco := preco + 15/100*preco
 else if (venda >= 1200) or (preco >= 80) then
 novo_preco := preco - 20/100*preco;

 writeln('O novo preco eh', novo_preco);
 end.